src/Workspaces/Remote/ServiceHub/Services/RemoteHostService.cs

```diff
@@ -205,8 +207,10 @@ private void SetSessionInfo(Dictionary<string, object> m)
             m["InstanceId"] = _primaryInstance;
         }
 
-        private static void SetGlobalContext(int uiCultureLCID, int cultureLCID, string serializedSession)
+        private void SetGlobalContext(int uiCultureLCID, int cultureLCID, string serializedSession)
         {
+            const int ReportIntervalInMinutes = 2;
```

Comment:
:bulb: Use `TimeSpan`

Reply:
can't since it is const. it does use timespan when actually used. TimeSpan.FromMinutes(ReportIntervalInMinutes)

Reply:
just made it readonly static